src/app/(main)/wines/page.tsx

```
@@ -33,19 +33,24 @@ const WinesPage = () => {
 
     console.log('data:', data);
   };
-
+  const [rating, setRating] = useState(4);
   return (
     <main className='mx-auto overflow-hidden px-[1.6rem] pb-[4.5rem]'>
       <h1 className='sr-only'>와인 목록 페이지</h1>
       <MonthlyWineSection />
-      <button onClick={() => open()}>Open Modal</button>
-      <Modal title='test' showCloseButton>
+      <button onClick={() => open('test01')}>Open Modal 01</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)